mitmproxy/proxy/mode_specs.py

```diff
@@ -223,17 +223,15 @@ class ReverseMode(ProxyMode):
 
     description = "reverse proxy"
     transport_protocol = TCP
-    scheme: Literal[
-        "http", "https", "http3", "tls", "dtls", "tcp", "udp", "dns", "quic"
-    ]
+    scheme: Literal["http", "https", "tls", "dtls", "tcp", "udp", "dns", "quic"]
```

Comment:
For reverse proxy mode, does it maybe make sense to keep `http3`? This way a user can explicitly specify HTTP/3 only. `quic://` forces raw QUIC instead.

(I wouldn't mind adding `http1://` and `http2://` eventually)

Reply:
Sounds good.